src/photon/photon.h

```diff
@@ -84,6 +81,10 @@ typedef struct {
    *  no task is running on the worker, this will be NULL. This is used to
    *  update the task table. */
   task *task_in_progress;
+  /** The process ID of the client. */
```

Comment:
Maybe document that if this is 0, that means that the client has not connected to the local scheduler yet.